packages/api/src/handlers/application/get-application-by-application-id.js

```diff
@@ -7,14 +7,15 @@ const { cache } = REDIS
 
 export default async (context, req, h) => {
   try {
-    const result = await checkCache(req)
+    if (req.query?.nocache !== 'true') {
```

Comment:
I think we want the default behaviour to be caching so that if there is no query string it should cache, maybe something along the lines of

`!req.query.nocache || req.query.nocache === 'false'`

Reply:
I think we're doing the same thing, but mine is the inverse logic version of yours where your condition is saying its equal to 'false' and I'm saying it's not equal to 'true'. Both cases will get the value from the cache by default, but your way is clearer so I will change it.

I don't want you to think I got the logic completely wrong. There is method to my madness. Lol

Reply:
Will also add the nocache to the summary page

Reply:
Thinking about (1) a bit more. If we're checking that `!req.query.nocache || req.query.nocache === 'false'`, won't that allow any other value that is truthy to be satisfy the condition. For example

nocache=blah

is not false, and not true, but will be truthy enough to go through. Maybe it doesn't matter, but again wanted to show why I did it the way I did. It ensures anything other than `nocache=true` will always default to it being loaded from the cache.